### PR TITLE
headrev spawn music

### PR DESCRIPTION
**Pull request by JoeHammad1844**
## About the PR

revs now get some unique spawn music when chosen

TODO:
- [x] find out where to define the headrev spawn sound so this actually works
- [x] ask the guy on youtube if this is okay and get a response
## Why / Balance

-10 confusion
+50 cool music

## Media

permission acquired
image

sample taken from
https://github.com/A-Guy173/Music/blob/main/revolution.mp3
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl: JoeHammad
- add: Headrevs now have their own music on spawn, credits to A-Guy on youtube

### CODE REVIEW

**Comment by JoeHammad1844** [comment]
it's ready

File: Resources/Audio/Ambience/Antag/attributions.yml [hunk 1]
```diff
@@ -9,4 +9,8 @@
 - files: ["zombie_start.ogg"]
   license: "CC-BY-NC-SA-3.0"
   copyright: "Made by @MIXnikita#1474 (Discord) for SS14"
-  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
\ No newline at end of file
+  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
+- files:
+  lisence: "CC0 1.0 Universal"
```
Comment (deltanedas) [inline Resources/Audio/Ambience/Antag/attributions.yml:14]: license spelled wrong + its CC0-1.0

Comment (JoeHammad1844) [reply Resources/Audio/Ambience/Antag/attributions.yml:14]: i copy pasted it from the artists github but i will change that

Comment (deltanedas) [reply Resources/Audio/Ambience/Antag/attributions.yml:14]: license as in the word license you spelled it lisence :trollface:

File: Resources/Audio/Ambience/Antag/attributions.yml [hunk 2]
```diff
@@ -11,6 +11,6 @@
   copyright: "Made by @MIXnikita#1474 (Discord) for SS14"
   source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
 - files:
-  license: "CC0 1.0 Universal"
+  license: "CC0-1.0 Universal"
```
Comment (deltanedas) [inline Resources/Audio/Ambience/Antag/attributions.yml:14]: no ` universal` either pro

**Comment by JoeHammad1844** [comment]
image
im smart

**Comment by deltanedas** [comment]
yeah you didnt change that line idk why its having a stroke

**Comment by JoeHammad1844** [comment]
i swear to god i typed it up before but it disappeared

**Comment by KingFroozy** [comment]
I think, it could be even better, if you'd add a muffled flash sound over the beginning of the audio.

**Comment by JoeHammad1844** [comment]
nah, doesnt sound good to me

File: Resources/Audio/Ambience/Antag/attributions.yml [hunk 3]
```diff
@@ -9,4 +9,8 @@
 - files: ["zombie_start.ogg"]
   license: "CC-BY-NC-SA-3.0"
   copyright: "Made by @MIXnikita#1474 (Discord) for SS14"
-  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
\ No newline at end of file
+  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
+- files: ["headrev_start.ogg"]
+  license: "CC0-1.0"
+  copyright: "Made by https://www.youtube.com/@a-guy173"
+  source: https://www.youtube.com/watch?v=S4lhWeI5d5o
```
Comment (metalgearsloth) [inline Resources/Audio/Ambience/Antag/attributions.yml:16]: This license is not valid:

1. No express license is given (that I can see) so you can't infer one you want.
2. It doesn't actually link to consent anywhere.

Comment (JoeHammad1844) [reply Resources/Audio/Ambience/Antag/attributions.yml:16]: the artist has a github page and in that was that lisence

Comment (deltanedas) [reply Resources/Audio/Ambience/Antag/attributions.yml:16]: yes the github repo has a cc0 license i confirm

Comment (JoeHammad1844) [reply Resources/Audio/Ambience/Antag/attributions.yml:16]: https://github.com/A-Guy173/Music/blob/main/LICENSE

Comment (JoeHammad1844) [reply Resources/Audio/Ambience/Antag/attributions.yml:16]: i also got explicit permission from the artist

Comment (VasilisThePikachu) [reply Resources/Audio/Ambience/Antag/attributions.yml:16]: You probably wanna link to the github file then. Not the youtube link

Comment (JoeHammad1844) [reply Resources/Audio/Ambience/Antag/attributions.yml:16]: linked

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
still links the yt, not the gh instance.

**Comment by JoeHammad1844** [comment]
> still links the yt, not the gh instance.

oh i had some confusion there

**Comment by VasilisThePikachu** [comment]
Just letting you know you have not made the requested changes. Did you forget to push?

**Review by EmoGarbage404 — APPROVED** [review]

**Pull request merged**